### ISSUE TITLE
Support getting publication information

### ISSUE THREAD

**Issue by Rayport** [enginebai/PyMedium#11]
Given a publication name, support to get publication information.

**Comment by Florents-Tselai** [comment]
Isn't this already supported by GET /{publication_name} ?
https://github.com/enginebai/PyMedium#publication

**Comment by enginebai** [comment]
Yes, now it supported either `GET /@{username}` or `GET `/{publication_name}`. (Notice there is a `@` prefix symbol.
